akka-http/src/main/scala/akka/http/scaladsl/server/RequestContextImpl.scala

```diff
@@ -23,17 +23,18 @@ import scala.concurrent.{ ExecutionContextExecutor, Future }
 private[http] class RequestContextImpl(
   val request:          HttpRequest,
   val unmatchedPath:    Uri.Path,
+  val tokenValues:      Map[ExtractionToken[_], Any],
```

Comment:
This could be a request attribute as well once they are managed but since it is specific to routing maybe it fits here well.